### DISCUSSION TITLE
Ubuntu 22 not supported?

### DISCUSSION THREAD

**Discussion by IvanLlanas** [azerothcore/azerothcore-wotlk#11541] [General]
While compiling on Ubuntu 21 only throws a bunch of warnings and works great, on Ubuntu 22 got this error (at 99%! xP):
![Clipboard 1](https://user-images.githubusercontent.com/25532326/165473841-4bbdcd5b-b571-4a4f-9e16-62ceaf89983a.jpg)
Maybe version 22 requires an additional pack not specified in the documentation?

**Comment by Kitzunu** [comment]
OpenSSL 3.0 not supported.

But also, only Ubuntu 20.04 is supported

**Reply by IvanLlanas** [reply]
So, it's a matter of Canonical repositories, isn't it?

**Reply by Kitzunu** [reply]
I dont know whar that means. But to me it looks like you are using OpenSSL 3.0 dont do that

**Reply by IvanLlanas** [reply]
Right :(
![Clipboard 2](https://user-images.githubusercontent.com/25532326/165693673-1ef8b488-a6c8-4155-a02f-16660ee28286.jpg)

**Reply by IvanLlanas** [reply]
Thanks for the info. =)
